lib/runc_sandbox.ml

```diff
@@ -308,9 +367,28 @@ let clean_runc dir =
       Os.sudo ["runc"; "--root"; dir; "delete"; item]
     )
 
-let create ?(fast_sync=false) ~runc_state_dir () =
-  Os.ensure_dir runc_state_dir;
-  let arches = get_arches () in
-  Log.info (fun f -> f "Architectures for multi-arch system: %a" Fmt.(Dump.list string) arches);
-  clean_runc runc_state_dir >|= fun () ->
-  { runc_state_dir; fast_sync; arches }
+let create ?state_dir (c : config) =
+  match state_dir with 
+    | None -> Fmt.failwith "Runc requires a state directory"
+    | Some runc_state_dir -> 
+        Os.ensure_dir runc_state_dir;
+        let arches = get_arches () in
+        Log.info (fun f -> f "Architectures for multi-arch system: %a" Fmt.(Dump.list string) arches);
+        clean_runc runc_state_dir >|= fun () -> 
+        { runc_state_dir; fast_sync = c.fast_sync; arches }
+
+open Cmdliner 
+
+let fast_sync =
+  Arg.value @@
+  Arg.opt Arg.bool false @@
+  Arg.info
+    ~doc:"Install a seccomp filter that skips allsync syscalls"
```

Comment:
```suggestion
    ~doc:"Install a seccomp filter that skips all synchronous syscalls"
```

Reply:
But the new text is wrong! The option skips all `sync` and related syscalls, not all synchronous syscalls.